src/main/app/claims/responseModelConverter.ts

```diff
@@ -287,16 +286,28 @@ export class ResponseModelConverter {
     return party
   }
 
-  private static convertPaymentIntention (paymentIntention: PaymentIntentionDraft): PaymentIntention {
-    return {
-      paymentOption: paymentIntention.paymentOption.option.value as PaymentOption,
-      paymentDate: this.convertPaymentDate(paymentIntention.paymentOption, paymentIntention.paymentDate),
-      repaymentPlan: paymentIntention.paymentPlan && {
-        instalmentAmount: paymentIntention.paymentPlan.instalmentAmount,
-        firstPaymentDate: paymentIntention.paymentPlan.firstPaymentDate.toMoment(),
-        paymentSchedule: paymentIntention.paymentPlan.paymentSchedule.value as PaymentSchedule
-      }
-    } as PaymentIntention
+  private static convertPaymentIntention (draft: ResponseDraft): PaymentIntention {
+    let paymentIntention
+    if (draft.isResponsePartiallyAdmitted()) {
+      paymentIntention = draft.partialAdmission.paymentIntention
+    } else {
+      paymentIntention = draft.fullAdmission.paymentIntention
+    }
+
+    if (draft.isResponseFullyAdmitted()
+      || !draft.isResponsePartiallyAdmittedAndAlreadyPaidAndPaymentIntentionPresent()) {
+      return {
+        paymentOption: paymentIntention.paymentOption.option.value as PaymentOption,
+        paymentDate: this.convertPaymentDate(paymentIntention.paymentOption, paymentIntention.paymentDate),
+        repaymentPlan: paymentIntention.paymentPlan && {
+          instalmentAmount: paymentIntention.paymentPlan.instalmentAmount,
+          firstPaymentDate: paymentIntention.paymentPlan.firstPaymentDate.toMoment(),
+          paymentSchedule: paymentIntention.paymentPlan.paymentSchedule.value as PaymentSchedule
+        }
+      } as PaymentIntention
+    } else {
+      throw new Error('Payment Intention and stating already paid cannot coexist')
```

Comment:
As discussed, Please update the data model with the new valid values so the user can then progress their defence.